app/forms/liability/LiabilityWeight.scala

```diff
@@ -70,4 +70,15 @@ object LiabilityWeight {
   def toForm(liabilityWeight: LiabilityWeight): Option[Option[String]] =
     Some(liabilityWeight.totalKg.map(_.toString))
 
+  def stripSpacesAndLetters(weight: String) = {
+    val extractNumberRegex = """^[^\d-]*(-?\d+\.?\d*)\D*$""".r
```

Comment:
what's the point of the ^[^\d-]* part?

Reply:
I guess it is for not matching such number 123YUG124